src/commands/models/detectorAppLens.ts

```diff
@@ -0,0 +1,17 @@
+export class AppLensARMresponse {
+    id: String;
+    name: String;
+    location: String;
+    resourceGroup: String;
+    properties: any;
+    type: string;
```

Comment:
Nitpick: inconsistent `String` vs `string`.  (I usually use the lower case version.)